appcues/src/main/java/com/appcues/analytics/AnalyticsTrackerExt.kt

```diff
@@ -23,19 +25,19 @@ internal fun AnalyticsTracker.track(experiment: Experiment) {
     )
 }
 
-internal fun AnalyticsTracker.trackExperienceError(experience: Experience, message: String) {
-    val errorId = UUID.randomUUID()
+internal fun AnalyticsTracker.trackRecoverableExperienceError(experience: Experience, message: String) {
+    // only track the recoverable error once
+    if (experience.renderErrorId != null) return
```

Comment:
the key check is here - don't track if it's already been tracked.

without this, it will keep re-tracking every 10 secs as the previous error gets back a qualify response, which then triggers looking through the cache and retrying the embeds, and failing again (loop)